docs/release-notes/7561-rp-workspace-mapping.rst

```diff
@@ -0,0 +1,8 @@
+:orphan:
+
+**New Features**
+
+-  Cluster: Adds RP-Workspace mapping feature, which allows resource pools to be bound to specific
+   workspaces. Bound resource pools can only be used by the workspaces they are bound to. When
+   combined with RBAC, this feature restricts the resources that users can access based on the
+   workspaces they have permissions to.
```

Comment:
-  Cluster: Add resource pool to workspace mapping feature, allowing resource pools to be bound to specific
   workspaces. Bound resource pools can only be used by the workspaces they are bound to. When
   combined with RBAC, this feature restricts the resources that users can access based on the
   workspaces they have permissions to.